src/platform/intel/cavs/platform.c

```diff
@@ -312,32 +315,35 @@ static struct spi_platform_data spi = {
 struct timer *platform_timer =
 	&platform_generic_queue[PLATFORM_MASTER_CORE_ID].timer;
 
+#if defined(CONFIG_SPI)
 int platform_boot_complete(uint32_t boot_message)
 {
-#if defined(CONFIG_SUECREEK)
 	return spi_push(spi_get(SOF_SPI_INTEL_SLAVE), &ready, sizeof(ready));
-#endif
-
+}
+#else
+int platform_boot_complete(uint32_t boot_message)
+{
 	mailbox_dspbox_write(0, &ready, sizeof(ready));
 #if defined(CONFIG_MEM_WND)
 	mailbox_dspbox_write(sizeof(ready), &sram_window,
 		sram_window.ext_hdr.hdr.size);
 #endif // defined(CONFIG_MEM_WND)
 
-#if defined(CONFIG_APOLLOLAKE)
+#if CAVS_VERSION == CAVS_VERSION_1_5
```

Comment:
Another one to be separated from new CONFIG_... options. In general, please move all CAVS_VERSION related changes to a single, separate commit.

Reply:
I've moved CAVS_VERSION defines change to another commit